articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+
+#### Linear Topology
+This type of topology also goes by the name **Bus Topology**. The name itself states that, *the nodes or components of a system are connected in a linear manner*. All the nodes are connected to one bus(cable) in which the data runs linearly from the start point to the endpoint. Terminators are attached at both ends to define the length of the bus. All the nodes are connected to the bus within the two terminators. So there is a *bidirectional* flow of data.
+The arrangement of bus topology is very simple and direct. Since only one transmission medium is used, the data is easily accessible. Due to its simple architecture, the extension of bus topology becomes easier. Since a single cable is used, when two workstations send data simultaneously they may hit one another. This leads to an increase in network traffic. Also, adding new devices and components may slow down the data transfer rate.
+
+![Linear Topology](/engineering-education/network-types-topologies/bus.jpg)
+
+#### Ring Topology
+Ring topology is a special kind of bus topology. *A linear topology in which both ends are joined together forms a **Ring Topology***. Thus no terminators are used and so the data flow is *unidirectional*. Because of that, for the data to reach from the first node to the last  it has to traverse through all the nodes. Since the connection of nodes is continuous, data flows infinitely throughout the topology until one node removes it.
+Like linear topology, the structure and arrangement of nodes are very simple. Due to its compact architecture, the configuration of nodes becomes easier. Since the nodes are connected in a loop, unpredictable troubleshooting problems may appear. To repair or fix one crashed node, the whole system has to be taken down. It is difficult to shut down one node and keep the rest of the system functional. Like the linear topology, adding new devices and components may slow down the data transfer rate.
+
+![Ring Topology](/engineering-education/network-types-topologies/ring.jpg)
+
+#### Mesh Topology
+Mesh topology is the solution to the problem with ring topology. Unlike ring topology, here *every node is connected with every other node present in the network*. In the case of a single node failure, the rest of the system can still be functional. Since one node connects many other nodes, data is shared through any one of the many paths available. This makes mesh arrangement a suitable network for industries and business establishments. A mesh topology in which all the nodes are connected to each other is called full connectivity, and the mesh topology in which only some nodes are connected to each other is called partial connectivity.

```

Comment:
very nice section as well - I think maybe rearranging some sentences in an advantage/disadvantage format - doesn't have to be a list - the sentence can simply state:
Mesh topology can come with many benefits such as....
"Since this increases power consumption to a larger extent...it can be seen as a disadvantage to the Mesh topology."
"Some organizations may see the "proper planning and requires high maintenance" as a disadvantage. 
Something around these line - feel free to modify any way you see fit.

We just want the reader to be able to clearly understand the facts stated .

Reply:
Restructured the paragraph. I felt the advantage/disadvantage(A/D) format as a list is more helpful and clearer to differentiate between the points.